### PR TITLE
Add guidance for supported string lengths

### PR DESCRIPTION
**Pull request by mattmcneeney**
See #509

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -321,6 +321,10 @@ Instances of these plans. For example, Platforms might restrict the actions
 that users can perform on existing Service Instances if the associated service
 or plan is deleted. Consider your deprecation strategy.
 
+Platforms MAY have limits on the length of strings that they can handle or
+display to end users, such as the description of a Service or Service Plan. It
+is RECOMMENDED that strings do not exceed 255 characters to increase the
+likelihood of having compatibility with any Platform.
```
Comment (n3wscott) [inline spec.md]: Sorry to sandbag one more comment, but perhaps we want to give an out to the platform for truncation? like:

`Platforms MAY truncate strings longer than 255 characters.` or something like that. This says the platforms have limits as a warning to the broker but does not say what the platform will do to a string from a broker that is too long.

**Comment by mattmcneeney** [comment]
@n3wscott Ha I didn't even realise I was doing that... Serves me right for editing in the browser!

**Comment by zrob** [comment]
lgtm

**Comment by fmui** [comment]
LGTM

**Pull request merged**